### DISCUSSION TITLE
Your Feedback is Valued

### DISCUSSION THREAD

**Discussion by TobiasPSP** [TobiasPSP/doneland_sources#1] [Announcements]
Feel free to submit informal ideas and requests or ask questions.

Let's focus on practical aspects and indulge into edge-case discussions or academic terminology. Let's all be guided by what is *helpful* and provides *good use*.

Very much looking forward to your feedback and comments!

**Comment by gitcandies** [comment]
looking at LM5116, but need to use it for  **constant current approx 8 amps** with  no load Vout 24V,  Can this be done?  What is the wiring change?  You mention it in your project, but the diagram is missing.
